packages/component-base/test/helpers.js

```diff
@@ -0,0 +1,38 @@
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+import { html, LitElement } from 'lit';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { ControllerMixin } from '../src/controller-mixin.js';
+import { PolylitMixin } from '../src/polylit-mixin.js';
+
+export const define = {
```

Comment:
This helper was missing from `@vaadin/component-base` - let's add it for testing mixins with Lit.